pages/osx/security.md

```diff
@@ -22,3 +22,11 @@
 - Add a certificate from file to a [k]eychain (if -k isn't specified, the default keychain is used):
 
 `security add-certificates -k {{keychain.name}} {{path/to/file.pem}}`
+
+- Add a CA certificate to the per-user Trust Settings:
+
+`security add-trusted-cert -k {{path/to/login.keychain-db}} {{ca-cert.pem}}`
```

Comment:
Does the last option `ca-cert.pem` just require the file name or a whole path?

(Also I just saw that this page could need a refactor for its arguments. E.g. `path` → `path/to/stuff`, `keychain.name` → `path/to/keychain.name`. Just in case you're motivated.)

Reply:
The path to `ca-cert.pem` does need to be the whole path, will amend the example. Will also refactor the arguments accordingly!

Also, throwing a quick thanks - I've been using tldr for coming up on 7 years, and they've been incredibly helpful. I'm equally delighted that contributing improvements is a really positive experience, y'all are great!

Reply:
I'm glad it's helpful to you! (And wow, 7 years ago `tldr` was still really small)